Oracle.md

```diff
@@ -189,20 +185,19 @@ The TLV serialization of oracle attestations is as follows.
     * [`string`:`outcome_n`]
 
 Where the signatures are ordered the same as the nonces in their original `oracle_event`.
-The outcomes should be the message signed, ordered the same as the signatures.
+The outcomes should be ordered the same as the signatures.
 
 ## Signing Algorithm
 
-Signatures should be generated following the algorithm specified in [BIP 340](https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki).
-Tagged hashes should also be used as defined in the [design section of BIP 340](https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki#Design).
+Signatures should be generated following the algorithm described in this section.
 
 The secret key (`sk`) should be the private key corresponding to the oracle's public key.
 
-The algorithm `Sign(sk, message, tag)` is defined as:
+The algorithm `Sign(sk, ci, r)` is defined as:
 
-* Let H = `tag_hash("DLC/oracle/" || tag)`
-* Let `m` = H(`message`)
-* Return `BIP340_sign(sk, m)`
+* Let `ci` the index of the outcome in the ordered list of possible outcomes that could be attested by the oracle. This list is either explicitly given by the oracle in the case of enumerated outcomes, or implicitly in the case of numerical decomposition.
+* Let `r` the pre-image of `R` announced by the oracle,
+* Return `s = r + ci * x`
```

Comment:
probably worth a link out to Lloyd's paper here or some other place.

Reply:
Added